pkg/controller/clusterdeployment/installconfigvalidation.go

```diff
@@ -66,6 +68,15 @@ func ValidateInstallConfig(cd *hivev1.ClusterDeployment, installConfigSecret *co
 				ic.Platform.VSphere.VCenters[0].Password == "") {
 			return ic, errors.New(missingvSphereCredentialsErr)
 		}
+	case platform.Nutanix != nil:
+		if ic.Platform.Nutanix == nil {
+			return ic, errors.New(noNutanixPlatformErr)
+		}
+		if ic.Platform.Nutanix.PrismCentral.Endpoint.Address != cd.Spec.Platform.Nutanix.PrismCentral.Address ||
+			ic.Platform.Nutanix.PrismCentral.Endpoint.Port != cd.Spec.Platform.Nutanix.PrismCentral.Port {
```

Comment:
nit: We've already extracted cd.Spec.Platform
```suggestion
		if ic.Platform.Nutanix.PrismCentral.Endpoint.Address != platform.Nutanix.PrismCentral.Address ||
			ic.Platform.Nutanix.PrismCentral.Endpoint.Port != platform.Nutanix.PrismCentral.Port {
```